docs/how-to-get-help.md

```diff
@@ -1,15 +1,65 @@
-## Onboarders
+## Co-pilot programme
 
-When your project has been approved by NHS England, we will agree an intensive support period with you when our experienced researchers and engineers will be made available to get you started.
+When your project has been approved by NHS England,
+we will agree with you an intensive programme of support with our experienced researchers, or *co-pilots*.
+You can find out more about the co-pilot programme [on our blog][the co-pilot programme].
 
-## Q&A Forum
+[the co-pilot programme]: https://www.bennett.ox.ac.uk/blog/2021/08/opensafely-co-pilot-programme-assisting-users-on-their-opensafely-journey/
 
-If you're outside the onboarding support period, or you're just trying OpenSAFELY out, your first port of call should be our [Q&A forum](https://github.com/opensafely/documentation/discussions) in GitHub. Before asking for help here, please search the [Q&A forum](https://github.com/opensafely/documentation/discussions) and these documentation pages to check that your query hasn’t already been addressed. Some of the most common issues that users face when using OpenSAFELY are listed in the [FAQ page](https://github.com/opensafely/documentation/discussions/445) on the Q&A.
+## Q&A forum
+
+If you're outside the co-pilot programme,
+then your first port of call should be to ask your question in the [Q&A forum][].
+Before asking your question,
+please search the [Q&A forum][] and the OpenSAFELY documentation,
+to check that your question hasn’t already been answered.
+
+[Q&A forum]: https://github.com/opensafely/documentation/discussions
+
+## Slack
+
+You will be invited to join the Bennett Institute's workspace during the co-pilot programme.
+If you're outside the co-pilot programme,
+then your second port of call should be to ask your question in any channel to which you have access.
+(Your first port of call should be to ask your question in the [Q&A forum](#qa-forum).)
```

Comment:
My rationale is to provide the reader with progressive context: they're in the Slack section, so rather than take them to the Q&A forum itself, we take them to the section on the Q&A forum to learn more, and then to the Q&A forum itself. The risk is that we condition them to think that every link with the text "Q&A forum" points to the section on the Q&A forum and not to the Q&A forum itself.

So, my answer to your question is "Yes". Is yours "No"?

Reply:
OK, makes sense to me. I will change my answer to "Yes".